src/Cli/dotnet/Parser.cs

```diff
@@ -279,6 +279,9 @@ public override void Write(HelpContext context)
                 {
                     new DotnetFormatForwardingApp(helpArgs).Execute();
                 }
+                else if (command.Name.Equals(FsiCommandParser.GetCommand().Name)){
```

Comment:
```suggestion
                else if (command.Name.Equals(FsiCommandParser.GetCommand().Name))
                {
```

Reply:
Thanks for this! My kingdom for an explicit format step I could run before push.